sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LegacyServiceClient.cs

```diff
@@ -1526,6 +1544,24 @@ private static Legacy.JobManifestTasks CreateTasks(TextAnalyticsActions actions)
             {
                 tasks.SentimentAnalysisTasks = Transforms.ConvertFromAnalyzeSentimentActionsToLegacyTasks(actions.AnalyzeSentimentActions);
             }
+
+            // Not supported in legacy service.
+            if (actions.AnalyzeHealthcareEntitiesActions != null && actions.AnalyzeHealthcareEntitiesActions.Count > 0)
+            {
+                throw Validation.NotSupported(nameof(AnalyzeHealthcareEntitiesAction), TextAnalyticsClientOptions.ServiceVersion.V2022_05_01);
```

Comment:
Should this include the version the customer tried using?

Reply:
Does it add value? They should know what they passed in. Even as-is, this is an actionable message. Making it more wordy doesn't seem to add value to me.

Reply:
Yes - because many customers silently change versions on a package upgrade when the default of ClientOptions changes.